src/main/java/lotto/domain/LottoNumber.java

```diff
@@ -8,24 +10,58 @@
 
 import static java.util.stream.Collectors.toList;
 
-public class LottoNumber {
+public class LottoNumber implements Comparable<LottoNumber> {
     private static final int MIN_LOTTO_NUMBER = 1;
     private static final int MAX_LOTTO_NUMBER = 45;
+    private static final int SUBLIST_START_INDEX = 0;
+    private static final int SUBLIST_END_INDEX_DIFFERENCE = 1;
+
+    private int lottoNumber;
 
-    private static final List<Integer> numberRange = IntStream
+    private static final List<LottoNumber> numbersWithinRange = IntStream
             .rangeClosed(MIN_LOTTO_NUMBER, MAX_LOTTO_NUMBER)
-            .mapToObj(Integer::new)
+            .mapToObj(LottoNumber::new)
             .collect(toList());
 
-    static List<Integer> generate(int numberCount) {
-        Collections.shuffle(numberRange, new Random(System.currentTimeMillis()));
+    private LottoNumber(int lottoNumber) {
+        this.lottoNumber = lottoNumber;
+    }
+
+    public static int valueOf(LottoNumber lottoNumber) {
+        return lottoNumber.lottoNumber;
+    }
 
-        List<Integer> numbers = new ArrayList<>();
-        for (int i = 0; i < numberCount; i++) {
-            numbers.add(numberRange.get(i));
+    public static List<LottoNumber> getRandomListInstance(int size) {
+        Collections.shuffle(numbersWithinRange, new Random(System.currentTimeMillis()));
+
+        return new ArrayList<>(numbersWithinRange.subList(SUBLIST_START_INDEX, getLastIndex(size)));
```

Comment:
```suggestion
        return new ArrayList<>(numbersWithinRange.subList(SUBLIST_START_INDEX, size));
```
현재 로또 숫자가 5개이네요. 
이 메서드를 작성한 이유가 따로 있으실까요?

Reply:
앗 제가 subList 메서드의 마지막 인덱스 미만으로 return 되는 것으로 착각했네요ㅠㅠ
버그를 발견 해 주셔서 감사해요. 그만큼 테스트 코드가 커버가 안 된 것이겠네요😓
수정 해 보겠습니다 :)